### PR TITLE
Send firefox crash report if crashed

### PR DESCRIPTION
**Pull request by Zaoliang**
we have sometimes issue with crash and need to send crash report. 
see https://bugzilla.suse.com/show_bug.cgi?id=1205511 and 
https://progress.opensuse.org/issues/120753

### CODE REVIEW

**Comment by Zaoliang** [comment]
VRs:
http://openqa.suse.de/tests/10042164
https://openqa.opensuse.org/tests/2912419
https://openqa.opensuse.org/tests/2912427

File: tests/x11/firefox.pm [hunk 1]
```diff
@@ -16,12 +16,24 @@ use base "x11test";
...
 use warnings;
 use testapi;
-use version_utils 'is_tumbleweed';
+use version_utils qw(is_opensuse is_tumbleweed);
 use Utils::Architectures 'is_s390x';
 
 sub run() {
     my ($self) = shift;
 
+    # we have sometime issue with firefox crash, export setting of automatic sending of crash report
```
Comment (rfan1) [inline tests/x11/firefox.pm:25]: Did you miss `source ~/.profile` ? :)

Comment (Zaoliang) [reply tests/x11/firefox.pm:25]: yes, the file is not there, I checked :)

File: tests/x11/firefox.pm [hunk 2]
```diff
@@ -16,12 +16,24 @@ use base "x11test";
...
+    # save the minidump and then force the application to close, see bsc#1205511
+    if (is_opensuse) {
+        select_console('root-console');
+        assert_script_run("touch ~/.profile");
+        script_run("cat > ~/.profile <<EOF
+export MOZ_CRASHREPORTER_AUTO_SUBMIT=1
+export MOZ_CRASHREPORTER_SHUTDOWN=1
+EOF
+true");
```
Comment (pdostal) [inline tests/x11/firefox.pm:34]:
```suggestion
        assert_script_run('echo -en "MOZ_CRASHREPORTER_AUTO_SUBMIT=1\n MOZ_CRASHREPORTER_SHUTDOWN=1" > ~/.profile');
```

File: tests/x11/firefox.pm [hunk 3]
```diff
@@ -16,12 +16,24 @@ use base "x11test";
...
+use version_utils qw(is_opensuse is_tumbleweed);
 use Utils::Architectures 'is_s390x';
 
 sub run() {
     my ($self) = shift;
 
+    # we have sometime issue with firefox crash, export setting of automatic sending of crash report
+    # save the minidump and then force the application to close, see bsc#1205511
+    if (is_opensuse) {
+        select_console('root-console');
```
Comment (pdostal) [inline tests/x11/firefox.pm:28]:
```suggestion
        select_serial_terminal;
```

**Review by pdostal — CHANGES_REQUESTED** [review]

File: tests/x11/firefox.pm (hunk 2 above)
Comment (Zaoliang) [reply tests/x11/firefox.pm:34]: thanks!

**Comment by Zaoliang** [comment]
https://openqa.opensuse.org/tests/2917528#step/firefox/25 looks good with latest change.

**Pull request merged**